src/navigation/image_database.cc

```diff
@@ -452,6 +455,21 @@ ImageDatabase::getFilename(const Vector3<millimeter_t> &position,
     return ss.str();
 }
 
+bool
+ImageDatabase::fileNameCompare(const std::string &fn1, const std::string &fn2)
+{
+    std::regex regex{ R"(^([^0-9]*)([0-9]+)\.(.+)$)" };
```

Comment:
some comments here would be nice - I find nothing is quite as confusing as last month's regex 😃

Reply:
Good point! I also swapped around the logic of the function to make it clearer.